sw/simulator/nps/nps_main_hitl.c

```diff
@@ -214,9 +214,24 @@ void *nps_ap_data_loop(void *data __attribute__((unused)))
   struct pprz_transport pprz_tp_logger;
 
   pprz_transport_init(&pprz_tp_logger);
+  uint32_t rx_msgs = 0;
+  uint32_t rx_commands = 0;
+  uint32_t rx_motor_mixing = 0;
+  uint8_t show_stats = 1;
```

Comment:
Seems useful if show_stats was configurable from the airframe file.

Reply:
I know, this is just the bare minimum fix to give the user at least some feedback.